app/assets/javascripts/edit_utils.js

```diff
@@ -279,6 +277,25 @@ $(() => {
     $(`#family_name_${suffix}`).val(familyName);
   }
 
+  // Fetch information via ORCID's public API and dumps the data into the elements indicated.
+  function fetchOrcid(orcidValue, givenNameId, familyNameId) {
+    if (isOrcid(orcidValue)) {
+      $.ajax({
+        url: `${pdc.orcid_url}/${orcidValue}`,
+        dataType: 'jsonp',
```

Comment:
Is there a reason not to request plain JSON?

Reply:
Two reasons: The code already had `jsonp` (this is not a new change) and if memory serves me well, back when we implemented this we noticed that we had to use `jsonp` for the Ajax call to work, something to do with the cross-domain calls.